tests/php/Forms/FormFieldTest.php

```diff
@@ -402,4 +402,30 @@ public function testLinkWithoutForm()
         $field = new FormField('Test');
         $field->Link('bar');
     }
+
+    /**
+     * @param string $name
+     * @param string $expected
+     * @dataProvider nameToLabelProvider
+     */
+    public function testNameToLabel($name, $expected)
+    {
+        $this->assertSame($expected, FormField::name_to_label($name));
+    }
+
+    /**
+     * @return array[]
+     */
+    public function nameToLabelProvider()
+    {
+        return [
+            ['TotalAmount', 'Total amount'],
+            ['Organisation.ZipCode', 'Organisation zip code'],
+            ['Organisation.zipCode', 'Organisation zip code'],
+            ['FooBarBaz', 'Foo bar baz'],
+            ['URLSegment', 'URL segment'],
```

Comment:
❤️

Reply:
The `testSchemaValidation` error is real - `SecurityID` should become `Security ID` - yay for tests!